SDL/Event.h

```diff
@@ -84,10 +86,27 @@ namespace SDL {
     int* superbinCPU;
     int8_t* pixelTypeCPU;
 
+    // Stuff that used to be global
+    const uint16_t nModules;
+    const uint16_t nLowerModules;
+    const unsigned int nPixels;
+    const std::shared_ptr<const modulesBuffer<Dev>> modulesBuffers;
+    const std::shared_ptr<const pixelMap> pixelMapping;
+    const std::shared_ptr<const EndcapGeometry<Dev, true>> endcapGeometry;
```

Comment:
CMSSW naming rules suggest using a trailing underscore for (private) data members (`m_` or `the` or some distrinct prefix/suffix are also acceptable). This makes the implementation more readable.
This PR could be a good opportunity to denote these are data members now

Reply:
That's a good point, thanks!